RefreshLocationThresholdsData/index.js

```diff
@@ -0,0 +1,40 @@
+const refresh = require('../Shared/csv-load/shared-refresh-csv-rows')
+const commonRefreshData = require('../Shared/csv-load/common-refresh-data')
+
+const functionSpecificData = [
+  { tableColumnName: 'LOCATION_ID', tableColumnType: 'NVarChar', expectedCSVKey: 'LOCATIONID' },
+  { tableColumnName: 'THRESHOLD_ID', tableColumnType: 'NVarChar', expectedCSVKey: 'ID' },
+  { tableColumnName: 'NAME', tableColumnType: 'NVarChar', expectedCSVKey: 'NAME' },
+  { tableColumnName: 'LABEL', tableColumnType: 'NVarChar', expectedCSVKey: 'LABEL' },
+  { tableColumnName: 'VALUE', tableColumnType: 'Decimal', expectedCSVKey: 'VALUE', precision: 38, scale: 8, nullValueOverride: true, preprocessor: commonRefreshData.returnNullForNaN },
+  { tableColumnName: 'FLUVIAL_TYPE', tableColumnType: 'NVarChar', expectedCSVKey: 'FLUVIALTYPE' },
+  { tableColumnName: 'COMMENT', tableColumnType: 'NVarChar', expectedCSVKey: 'COMMENT', nullValueOverride: true, preprocessor: returnNullForEmptyString },
+  { tableColumnName: 'DESCRIPTION', tableColumnType: 'NVarChar', expectedCSVKey: 'DESCRIPTION' }
+]
+
+module.exports = async function (context) {
+  const refreshData = {
+    csvUrl: process.env.LOCATION_THRESHOLDS_URL,
```

Comment:
Unsure of this URL, however it will need to be confirmed and added to the app environment before these changes can be deployed. I think Lindsay Ness will be able to find this one out.